src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -2094,6 +2095,14 @@ def load_arguments(self, _):
         c.argument("dns_zone_resource_ids", options_list=["--ids"], required=True)
         c.argument("attach_zones")
 
+    with self.argument_context('aks check-network outbound') as c:
+        c.argument('cluster_name', options_list=['--name', '-n'],
+                   required=True, help='Name of the managed cluster.')
+        c.argument('node_name', help='Name of the node to perform the connectivity check.')
+        c.argument('custom_endpoints',
+                   help='Additional endpoint(s) to perform the connectivity check, separated by comma.',
+                   validator=validate_custom_endpoints)
```

Comment:
If the value of this param is a list, it is better to use `nargs='+'` to define the param, and input would be separated by spaces automatically.

Reply:
Oh I see! Updated